frontend/src/metabase/nav/components/WhatsNewNotification/WhatsNewNotification.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback, useMemo } from "react";
+import { t } from "ttag";
+import { updateSetting } from "metabase/admin/settings/settings";
+import IconButtonWrapper from "metabase/components/IconButtonWrapper";
+import { Icon } from "metabase/core/components/Icon";
+import { color } from "metabase/lib/colors";
+import { useDispatch, useSelector } from "metabase/lib/redux";
+import { getIsEmbedded } from "metabase/selectors/embed";
+import { getSetting } from "metabase/selectors/settings";
+import { Anchor, Flex, Paper, Stack, Text } from "metabase/ui";
+import Sparkles from "./sparkles.svg?component";
+import { getLatestEligibleReleaseNotes } from "./utils";
+
+export function WhatsNewNotification() {
+  const dispatch = useDispatch();
+  const isEmbedded = useSelector(getIsEmbedded);
+  const versionInfo = useSelector(state => getSetting(state, "version-info"));
+  const currentVersion = useSelector(state => getSetting(state, "version"));
+  const lastAcknowledgedVersion = useSelector(state =>
+    getSetting(state, "last-acknowledged-version"),
+  );
+
+  const url: string | undefined = useMemo(() => {
+    const lastEligibleVersion = getLatestEligibleReleaseNotes({
+      versionInfo,
+      currentVersion: currentVersion.tag,
+      lastAcknowledgedVersion: lastAcknowledgedVersion,
+      isEmbedded,
+    });
+
+    return lastEligibleVersion?.announcement_url;
+  }, [versionInfo, currentVersion.tag, lastAcknowledgedVersion, isEmbedded]);
+
+  const dimiss = useCallback(() => {
+    dispatch(
+      updateSetting({
+        key: "last-acknowledged-version",
+        value: currentVersion.tag,
+      }),
+    );
+  }, [currentVersion.tag, dispatch]);
+
+  if (!url) {
+    return null;
+  }
+  return (
+    <Paper my="lg" mx="auto" p="md" shadow="md" withBorder w={244}>
+      <Stack spacing="sm">
+        <Flex justify="space-between">
+          <Sparkles color={color("brand")} />
+          <IconButtonWrapper onClick={dimiss}>
+            <Icon name="close" color={color("bg-dark")} />
```

Comment:
Let's have this darken on hover to make it feel interactive. We can use `text-medium` for the hover state color (I know the names are silly, we're working on it).